packages/firestore/src/local/index_manager.ts

```diff
@@ -103,13 +103,12 @@ export interface IndexManager {
 
   /**
    * Returns the documents that match the given target based on the provided
-   * index.
+   * index. Returns `null` if the target does not have a matching index.
```

Comment:
This change is pending review on Android: https://github.com/firebase/firebase-android-sdk/pull/3444